vscode-dotnet-runtime-library/src/Acquisition/AcquisitionInvoker.ts

```diff
@@ -118,7 +118,7 @@ If you cannot change this flag, try setting a custom existingDotnetPath via the
                                 reject(newError);
                             }
                         }
-                        else if (stderr && stderr.length > 0)
+                        else if (stderr?.length > 0)

```

Comment:
```suggestion
                        else if ((stderr?.length ?? 0) > 0)

```

This is correct previously too but this feels easier to read to me. Subjective.